languages/js/src/Oso.ts

```diff
@@ -244,22 +263,25 @@ export class Oso<
    * @returns A query that selects authorized resources of type `resourceCls`
    */
   async authorizedQuery(
-    actor: any,
-    action: any,
-    resourceCls: any
-  ): Promise<any> {
+    actor: Actor,
+    action: Action,
+    resourceCls: Class
```

Comment:
Should this be `Resource`? Or maybe a type param that is a `Resource` but also extends `Class`? Not sure how to express that

Reply:
Verified that this works: [`8d49c50` (#1211)](https://github.com/osohq/oso/pull/1211/commits/8d49c505a9d6c96f586d98f977b0836cc46e27d3#diff-2178a05ab0077a949c52ac5f9d679b82d59967f2a2536708262b202d53dffee6R268)